aixplain/factories/model_factory.py

```diff
@@ -55,7 +54,7 @@ def _create_model_from_response(cls, response: Dict) -> Model:
         """
         if "api_key" not in response:
             response["api_key"] = cls.api_key
-        return Model(response["id"], response["name"], supplier=response["supplier"]["id"], api_key=response["api_key"])
+        return Model(response["id"], response["name"], supplier=response["supplier"]["id"], api_key=response["api_key"], pricing=response["pricing"])
```

Comment:
Lucas, is the pricing information something that is optionally set within the model? The current params within the model's constructor do not seem to have this information.

Reply:
@krishnadurai the pricing information would be stored in Model's additional_info field. By calling get method, this would be an example response:
```
{'
  id': '60ddef908d38c51c5885dd1e',
 'name': 'Speech Recognition - English (United States)',
 'description': '',
 'supplier': 'AWS',
 'version': '1.0',
 'license': None,
 'privacy': <Privacy.PRIVATE: 'Private'>,
 'url': 'https://dev-models.aixplain.com/api/v1/execute',
 'backend_url': 'https://dev-platform-api.aixplain.com/',
 'api_key': 'fc0b527c87782bc23bb7bf65babfa7918b6bebc781289bfffeb28147486ab744',
 'additional_info': {'pricing': {'price': 0.024,
   'unitType': 'TIME',
   'unitTypeScale': 'MINUTE'}}
}
```